ci/check_es.py

```diff
@@ -0,0 +1,47 @@
+#!/usr/bin/env python
+#   Licensed under the Apache License, Version 2.0 (the "License");
+#   you may not use this file except in compliance with the License.
+#   You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+#   Unless required by applicable law or agreed to in writing, software
+#   distributed under the License is distributed on an "AS IS" BASIS,
+#   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#   See the License for the specific language governing permissions and
+#   limitations under the License.
+
+import argparse
+import elasticsearch
+import sys
+
+def _check_index(server,port,uuid,index):
+    es = elasticsearch.Elasticsearch([{'host': server,'port': port}])
+    results = es.search(index=index, body={'query': {'match': {'uuid': {'query': uuid, 'operator': 'AND'}}}})
```

Comment:
Just a suggestion( more like a fyi ) we should add a `size=1` in the search call, we don't really want to look at data ( not setting size would fetch us 10 documents by default ) this will make the query execution faster as well as reduce the load on ES

Reply:
If I don't use the and it becomes an implicit or and always returns true even if it doesn't find anything.